src/components/main-form.js

```diff
@@ -519,19 +546,93 @@ class MainForm extends React.Component {
                   and your return on investment lower. We factor all of this
                   automatically so the final result you see on your right is not
                   the actual money you will have gained/lost in{' '}
-                  {this.state.longerOption.term} years, but rather, the money
-                  you will have gained/lost in {this.state.longerOption.term}{' '}
-                  years adjusted to today’s purchasing power. If you’re curious
-                  about what the actual number would be, give inflation a value
-                  of 0%. But remember that everything will be a lot more
-                  expensive in {this.state.longerOption.term} years, and that is
-                  why we need to include inflation.
+                  {longerOption.term} years, but rather, the money you will have
+                  gained/lost in {longerOption.term} years adjusted to today’s
+                  purchasing power. If you’re curious about what the actual
+                  number would be, give inflation a value of 0%. But remember
+                  that everything will be a lot more expensive in{' '}
+                  {longerOption.term} years, and that is why we need to include
+                  inflation.
                 </Accordion>
               </FormGroup>
             </Form>
           </Card>
         </Col>
         <Col xs='8'>
+          <Summary className='p-4 mr-5'>
+            <h4>Explanation</h4>
+            <p>
+              <b>{formatMoney(this.state.optCost)}</b> is the difference between
+              your {shorterOption.term} year investment total (
+              <b>{formatMoney(shorterOption.fv)}</b>) and your{' '}
+              {longerOption.term} year investment total (
+              <b>{formatMoney(longerOption.fv)}</b>).
+            </p>
+            <Col>
+              <h5>{shorterOption.term} year scenario:</h5>
+              <ol>
+                <li>
+                  With the <b>{shorterOption.term} year mortgage</b> you pay{' '}
+                  <b>{formatMoney(shorterOption.pmt)}</b> monthly for{' '}
```

Comment:
Looks like this may need a sign flip

Reply:
yeah I was thinking about that. I'm wondering if it would be better to turn everything into positives and handle the signs only in calculations (as opposed to handling them in the UI). I think almost every number in the app should be positive, including the ones on the table. Negatives are more confusing to the average joe

Reply:
Negatives would be accurate in the accounting world, and when we do calculations internally, we probably still want them negative.

Keeping the positive seems like a presentation responsibility for the user, so I think we should flip the signs when we display